_pages/clients/python.md

```diff
@@ -19,28 +19,26 @@ redirect_from:
 
 {% include video-responsive.html height="560" width="315" url="https://www.youtube.com/embed/bZB2vu0v6A0" %}
 
-This guide provides a walk-through of how to use the official Algorithmia Python Client to call algorithms and manage data through the Algorithmia platform.
+The Algorithmia Python client provides a native Python interface to the Algorithmia API, letting developers manage and call algorithms, work with data in object stores using the Algorithmia Data Portal, and access other features of the Algorithmia platform.
 
-Here you will learn how to install the Algorithmia Python Client, work with the Data API by uploading and downloading files, create and update directories and permission types and last, you'll learn how to call an algorithm that summarizes text files.
+This guide will cover setting up the client, calling an algorithm using direct user input, calling an algorithm that accesses data through the Algorithmia Data Portal, and using Algorithmia's Hosted Data service. For complete details about the Algorithmia API, please refer to the [API Docs](/developers/api/?python#introduction).
```

Comment:
My understanding is that the "Data Portal" is a platform feature which allows you to access multiple object stores (Hosted Data, S3, etc.) using the same code and URIs, while "Data Sources" are the object stores which the Data Portal has been configured to use.

Reply:
Yeah, data portal has been used in the past, but since it's not in the UI anywhere (anymore), so it might be confusing to users if it only appears in the docs - gonna check with Hoover on this real quick

Reply:
Got confirmation to remove the terminology "Data Portal" and we can just use "Data Sources" since that's the header of the page that it links to